image/image.go

```diff
@@ -380,6 +405,15 @@ func bootstrapToRootDir(tsto *ToolingStore, model *asserts.Model, opts *Options,
 			fmt.Fprintf(Stdout, "Fetching %s\n", snapName)
 		}
 
+		dlOpts.Channel = opts.Channel
```

Comment:
as I wrote, I think at this point it would be cleaner to bring making dlOpts inside the loop:

    snapChannel := opts.Channel
    if name == model.Kernel() ... {
    }
    dlOpts := ...{
       ...
       Channel: snapChannel
    }